
For easy of comparing test results, always run in the same order and do not
automatically re-launch the failures in verbose mode.

diff -wpruN '--exclude=*.orig' a~/Tools/scripts/run_tests.py a/Tools/scripts/run_tests.py
--- a~/Tools/scripts/run_tests.py	1970-01-01 00:00:00
+++ a/Tools/scripts/run_tests.py	1970-01-01 00:00:00
@@ -37,8 +37,6 @@ def main(regrtest_args):
     args.extend(['-W', 'error::BytesWarning'])
 
     args.extend(['-m', 'test',    # Run the test suite
-                 '-r',            # Randomize test order
-                 '-w',            # Re-run failed tests in verbose mode
                  ])
     if sys.platform == 'win32':
         args.append('-n')         # Silence alerts under Windows
